src/trackers/helpers/sharedData.js

```diff
@@ -18,28 +17,37 @@ class SharedData {
         this.domains = _getJSON(`${build}/generated/domain_summary.json`) || {}
         this.abuseScores = _getJSON(`${build}/generated/api_fingerprint_weights.json`)
         this.categories = _getCategories()
-        if (fs.existsSync(`${cfg.trackerDataLoc}/entities`)) {
-            this.domainToEntity = _readEntities()
-            this.entityMap = entityHelper.entityMap(`${cfg.trackerDataLoc}/entities`)
```

Comment:
Both `entityHelper.entityMap` and `_readEntities` were reading entity files. I merged them into one function. `entityHelper` is not used anywhere outside of this file.